lib/vsc/filesystem/oceanstor.py

```diff
@@ -624,8 +638,8 @@ def get_fileset_info(self, filesystem_name, fileset_name):
             self.log.raiseException(errmsg, OceanStorOperationError)
 
         # Convert VSC fileset name to OceanStor ones
-        # - names of user folders grouped in filesets 100, 101,...
-        fileset_name = re.sub('^vsc([0-9]{3})$', r'\1', fileset_name)
+        for banned_fs in BANNED_FILESET_NAMES:
```

Comment:
are you doing a zero op here as you're looping for the regex? vsc123 > 123 -> vsc123?

Reply:
nope, `BANNED_FILESET_NAMES` has 2 levels, this only loops over the first one

Reply:
right, it's packed in a tuple.